packages/frontend/src/page/components/wallet-position/recover-reward.tsx

```diff
@@ -62,8 +60,8 @@ export const RecoverReward = ({
         loading: loadingEffectiveRewardBalances,
     } = useWatchKPITokenRewardBalances(kpiToken.address, rewards);
 
-    const { config: recoverConfig, isLoading: loadingRecoverConfig } =
-        usePrepareContractWrite({
+    const { data: recoverConfig, isLoading: loadingRecoverConfig } =
```

Comment:
Same as above

Reply:
Fixed in 9dd76fbb77f7325cf86cb2441a2ea4f52fafeea5.